tests/sys_fido2_ctap/main.c

```diff
@@ -10,27 +10,60 @@
  * @ingroup tests
  * @{
  * @file
- * @brief       FIDO2 CTAP test application that creates an authenticator
- *              which uses CTAPHID as underlying communication protocol
+ * @brief       FIDO2 CTAP test application that tests CTAP functionality
+ *              without transport layer.
  *
- * @author      Nils Ollrogge <nils-ollrogge@outlook.de>
+ * @author      Nils Ollrogge <nils.ollrogge@fu-berlin.de>
  * @}
  */
-
 #include <stdio.h>
 #include <stdlib.h>
 
-#define ENABLE_DEBUG    (0)
-#include "debug.h"
-
-#include "xtimer.h"
+#include "embUnit.h"
 
 #include "fido2/ctap.h"
-#include "fido2/ctap/transport/ctap_transport.h"
+
+static uint8_t mc_data[] = {0xa6, 0x01, 0x58, 0x20, 0x0c, 0x8b, 0x27, 0x47, 0x8e, 0x25, 0x63, 0xb2, 0xce, 0x21, 0x2d, 0x42, 0x81, 0xbc, 0x22, 0x52, 0xa8, 0x81, 0xc9, 0xa6, 0x56, 0x6e, 0xee, 0xf2, 0x62, 0xbd, 0x09, 0x4d, 0x1c, 0x3e, 0x73, 0x6e, 0x02, 0xa2, 0x62, 0x69, 0x64, 0x6b, 0x65, 0x78, 0x61, 0x6d, 0x70, 0x6c, 0x65, 0x2e, 0x63, 0x6f, 0x6d, 0x64, 0x6e, 0x61, 0x6d, 0x65, 0x6a, 0x45, 0x78, 0x61, 0x6d, 0x70, 0x6c, 0x65, 0x20, 0x52, 0x50, 0x03, 0xa2, 0x62, 0x69, 0x64, 0x47, 0x75, 0x73, 0x65, 0x72, 0x5f, 0x69, 0x64, 0x64, 0x6e, 0x61, 0x6d, 0x65, 0x67, 0x41, 0x2e, 0x20, 0x55, 0x73, 0x65, 0x72, 0x04, 0x84, 0xa2, 0x63, 0x61, 0x6c, 0x67, 0x26, 0x64, 0x74, 0x79, 0x70, 0x65, 0x6a, 0x70, 0x75, 0x62, 0x6c, 0x69, 0x63, 0x2d, 0x6b, 0x65, 0x79, 0xa2, 0x63, 0x61, 0x6c, 0x67, 0x27, 0x64, 0x74, 0x79, 0x70, 0x65, 0x6a, 0x70, 0x75, 0x62, 0x6c, 0x69, 0x63, 0x2d, 0x6b, 0x65, 0x79, 0xa2, 0x63, 0x61, 0x6c, 0x67, 0x38, 0x24, 0x64, 0x74, 0x79, 0x70, 0x65, 0x6a, 0x70, 0x75, 0x62, 0x6c, 0x69, 0x63, 0x2d, 0x6b, 0x65, 0x79, 0xa2, 0x63, 0x61, 0x6c, 0x67, 0x39, 0x01, 0x00, 0x64, 0x74, 0x79, 0x70, 0x65, 0x6a, 0x70, 0x75, 0x62, 0x6c, 0x69, 0x63, 0x2d, 0x6b, 0x65, 0x79, 0x08, 0x50, 0x31, 0xfc, 0xac, 0xe4, 0xe0, 0x2c, 0x93, 0x21, 0x92, 0xc0, 0xe9, 0x8e, 0xe6, 0x47, 0xf6, 0x30, 0x09, 0x01};
```

Comment:
It'd help to have this written in user readable form, ideally (but not necessarily) with a means to re-generate it (or a similar, if there's a random nonce in it).

Reply:
I generated this data with the [fido python](https://github.com/Yubico/python-fido2) lib. Would providing a python script that prints out this sequence of bytes suffice ?

Reply:
Yes. Doesn't even need to be byte-by-byte or automated, but a 1-2 line comment with the code to build it would both help users understand it, and help other maintainers fix things when they break.

Reply:
Done. Added a python script which will generate the array on input data.